src/modules/dashboard/components/FeaturedChannelListItem.tsx

```diff
@@ -0,0 +1,223 @@
+// React + Web3 Essentials
+import { FC, useMemo, useState, useEffect } from 'react';
+
+// External Packages
+import { useSelector } from 'react-redux';
+import { cloneDeep } from 'lodash';
+
+// Internal Components
+import { Box, Button, CaretDown, NotificationMobile, Skeleton, Text } from 'blocks';
+import SubscribeChannelDropdown from 'common/components/SubscribeChannelDropdown';
+import UnsubscribeChannelDropdown from 'common/components/UnsubscribeChannelDropdown';
+
+// Internal Configs
+import { ChannelDetailsProps } from '../configs/DashboardFeaturedChannels.config';
+import { useGetChannelDetails } from 'queries';
+import { css } from 'styled-components';
+import Ethereum from 'blocks/illustrations/components/Ethereum';
+import { LOGO_ALIAS_CHAIN } from '../configs/ChainDetails';
+import TickDecoratedCircleFilled from 'blocks/icons/components/TickDecoratedCircleFilled';
+import { Link } from 'react-router-dom';
+import VerifiedToolTipComponent from './VerifiedToolTipComponent';
+
+type FeaturedChannelListItemProps = {
+  channel: ChannelDetailsProps;
+};
+
+const FeaturedChannelListItem: FC<FeaturedChannelListItemProps> = (props) => {
+  const { channel } = props;
+
+  const { subscriptionStatus, userSettings: currentUserSettings } = useSelector((state) => state.channels);
+
+  const userSettings = useMemo(() => {
+    return cloneDeep(currentUserSettings);
+  }, [currentUserSettings]);
+
+  const formatSubscriberCount = (count: number) => {
+    if (count >= 1000000) {
+      return (count / 1000000).toFixed(1) + 'M';
+    } else if (count >= 1000) {
+      return (count / 1000).toFixed(1) + 'K';
+    } else {
+      return count;
+    }
+  };
+
+  const { data: channelDetails, isLoading } = useGetChannelDetails(channel.channel);
+
+  const [subscribed, setSubscribed] = useState(false);
+  const [subscriberCount, setSubscriberCount] = useState(0);
+
+  useEffect(() => {
+    if (channelDetails) {
+      setSubscribed(subscriptionStatus[channel.channel]);
+      setSubscriberCount(channelDetails.subscriber_count);
+    }
+  }, [subscriptionStatus, channelDetails]);
+
+  const [loading, setLoading] = useState(false);
+
+  const AliasChain = channelDetails?.alias_blockchain_id && LOGO_ALIAS_CHAIN[+channelDetails.alias_blockchain_id];
+
+  return (
+    <>
+      <Box
+        display="flex"
+        flexDirection="column"
+        border={{ light: '1px solid gray-200', dark: '1px solid gray-800' }}
+        padding="s6"
+        borderRadius="24px"
+        gap="s3"
+        width={{ initial: '27.68%' }}
+        css={css`
+          flex-shrink: 0;
+        `}
+      >
+        <Box
+          display="flex"
+          justifyContent="space-between"
+        >
+          <Skeleton isLoading={isLoading}>
+            <Box
+              width="40px"
+              height="40px"
+              borderRadius="var(--r3)"
+              css={css`
+                overflow: hidden;
+              `}
+            >
+              <img
```

Comment:
<img width="619" alt="Screenshot 2024-06-14 at 3 39 24 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/175c3b3b-992e-4e94-84f7-b2699c24acaf">

 @abhishek-01k 
The image size is 52 x 52 and there is also a border for the image. I think we need to remove the parent box component and then apply the css directly to img.